etc/ProdOfflineConfiguration.py

```diff
@@ -32,6 +32,9 @@
 # Set the verstion configuration (not used at the moment)
 setConfigVersion(tier0Config, "replace with real version")
 
+# replay these run
+#setInjectRuns(tier0Config, [ 289461 ])
```

Comment:
I think this was mostly intended as a reminder, but it might not be necessary now we have synced the replay config. @johnhcasallasl , what do you think?